inner-scripts/switch-space

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
```

Comment:
Why wouldn't users outside the core container also want to use this script?

Reply:
I removed switch-space.